doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                    own personal working copy and they can work simultaneously and independently 
+                    to modify their personal copies which can then be merged together later. 
+                    Note it is the humans reponsibility to merge the files correctly and 
+                    resolve any conflicts by manually choosing the correct solution after 
+                    usually after discussions with the other people editing involved.
+                </li>
+                <li>Each repository change which is committed to the repository creates a new 
+                    state of the filesystem tree, this is called a revision number.
+                </li>
+                <li>The revision number is applied to the entire repository tree and not 
+                    individual files and is therefore also known as the Global Revision Number.
+                    For example: at revision number N then a file may be the same as it was at 
+                    revision number M but a different file was changed between the two 
+                    revisions.
+                </li>
+                <br/>
+                Version Control information
+                <br/>
+                The <a href="http://svnbook.red-bean.com/">Red bean book</a> describes a 
+                version control system.
+                <br/>
+            </div>
+            </p>
+        </ul>
+        <h1><a id="S2">The basics of svn and where to learn more</a></h1>
+        <p>
+        <ul>
+            <div class="page-header">
+                <h3>Files</h3>
+            </div>
+            <div class="well">
+                <li>File sizes</li>
+                Large files and file that are constantly growing can be an issue. 
+                <br/>
+                Files greater than 10MB are not allowed to be committed to the repository 
+                without discussion with the Fcm team.
+                They may be permitted but a quick discussion to assess the need is required.
+                The problem with every growing files is that they are not best suited to a
+                version control repository and may be best located else where.
+                Note they can be in the same directory as your working/local copy but Fcm does 
+                not need to ever know about them.
+                <br/>
+                <br/>
+                <li>File types</li>
+                ASCII v binary files
+                <br/>
+                The main type of files that we will be committing to a repository are: 
+                ASCII and binary files.
+                <br/>
+                <ul>
+                    <li>Assci files can be incrementally updated where only the changed or new 
+                        part is added to the file, for example a txt file, code, LaTeX file.
+                    </li>
+                    <br/>
```

Comment:
Why another br here? Probably better to close and open the ul tags if you're just looking to increase the gap between the two points.